### ISSUE TITLE
Raise Max Secondaries Slider

### ISSUE THREAD

**Issue by Mako88** [dxx-rebirth/dxx-rebirth#123]
Since you can now duplicate up to x7, it would be very nice to allow you to hold up to 20 smarts/megas. There should be a slider that would allow raising the max cap of secondaries.

(I believe this was mentioned elsewhere, but I'm not sure where...)

**Comment by zicodxx** [comment]
I think actually this should be done by granting players the Ammo Rack item at start.

**Comment by Mako88** [comment]
What about for D1? And what are the maximum levels with the Ammo Rack?

**Comment by zicodxx** [comment]
The Ammo Rack should allow you to hold twice as much ammo as initially. With a slight modification, this item could also be granted in D1 as a startup item. It just doesn't exist physically in Descent 1 as a sprite.
Tho I would not go much higher than that. eventually the whole game balance will go out of the window I think. Otherwise we may just merge both these features into an "infinite ammo" setting methinks.

**Comment by Mako88** [comment]
Yeah, maybe just a [ ] Double Secondary Capacity option in the host menu would be good.
